lang/lang.pt-BR.json

```diff
@@ -881,41 +881,41 @@
         },
         "links": {
             "invite": "https://discord.com/oauth2/authorize?client_id=471091072546766849&scope=bot%20applications.commands&permissions=85056",
-            "inviteEmbed": "[Convide {{REF:bot.name}} para o seu Server!]({{REF:links.invite}})",
+            "inviteEmbed": "[Convide {{REF:bot.name}} para o seu servidor!]({{REF:links.invite}})",
             "docs": "https://docs.friendtime.ml",
-            "docsEmbed": "[Documentação]({{REF:links.docs}}/)",
+            "docsEmbed": "[Ver Documentação]({{REF:links.docs}}/)",
             "vote": "https://top.gg/bot/471091072546766849/vote",
-            "voteEmbed": "[Vote em mim {{REF:bot.name}}!]({{REF:links.vote}})",
+            "voteEmbed": "[Vote no {{REF:bot.name}}!]({{REF:links.vote}})",
             "support": "https://discord.gg/GQcBR8e",
-            "supportEmbed": "[Junte ao servidor de suporte]({{REF:links.support}})",
+            "supportEmbed": "[Junte-se ao servidor de suporte]({{REF:links.support}})",
             "stream": "https://www.twitch.tv/novakevin",
             "author": "https://github.com/KevinNovak",
             "authorEmbed": "[Kevin Novak]({{REF:links.author}})",
             "source": "https://github.com/KevinNovak/Friend-Time",
-            "sourceEmbed": "[Verificar o código]({{REF:links.source}})",
+            "sourceEmbed": "[Verificar o código-fonte]({{REF:links.source}})",
             "template": "https://github.com/KevinNovak/Discord-Bot-TypeScript-Template",
-            "templateEmbed": "[Discord Bot TypeScript Template]({{REF:links.template}})",
```

Comment:
"Discord Bot TypeScript Template" is the name of the project. I think this should probably be left as is.

Reply:
What about putting both names, the official name and the translation of the name?

Reply:
All of the other lang files have left it as is. I think I'd prefer it to be consistent.

Reply:
Done.